contrib/epee/include/net/abstract_tcp_server.h

```diff
@@ -38,6 +38,9 @@
 
 #pragma comment(lib, "Ws2_32.lib")
 
+#undef MONERO_DEFAULT_LOG_CATEGORY
+#define MONERO_DEFAULT_LOG_CATEGORY "net"
```

Comment:
You can do that in your code if you prefer, using, eg, MCERROR(cat,x) instead of MERROR(x)

It does seem wordy to me, but maybe it becomes less so when you get into the habit...